testing/test_helpconfig.py

```diff
@@ -38,6 +38,34 @@ def test_help(testdir):
     )
 
 
+def test_empty_help_param(testdir):
```

Comment:
You could join the two test cases you've written into one by [parametrizing them](https://docs.pytest.org/en/stable/parametrize.html)